content/updates/automation/2-Powerful-new-actions.md

```diff
@@ -0,0 +1,39 @@
+# Powerful new automated rule actions
+
+## Concatenate attribute values and free texts
+::: meta-data type="New" ee-only="true" link-to-doc="../articles/get-started-with-the-rules-engine.html#concatenate"
+:::
+
+Writing product titles and descriptions can take time, but that was before! With the new `concatenate` rule action, you can now combine your attributes, add free-text fields and line breaks, and set the resulting value in another attribute. Very handy when you want to automate the creation of attractive product descriptions and other product content.
+
+This can also be used to ensure your product information is always formatted the same way. For instance, with the creation of a rule to concatenate your product attributes `Name` + `Color` + `Size` and set this new value chain into your `Product title` attribute.
+
+Last but not least, to avoid any mistake, you can preview the created content when setting up your rule.
+
+![Example of an automated product description creation](../img/automated-product-description-with-rules.png)
+
+## Calculate attribute values
+::: meta-data type="New" ee-only="true" link-to-doc="../articles/get-started-with-the-rules-engine.html#calculate"
+:::
+
+From complex to simple calculations, the `calculate` rule action enables you to do mathematical operations to fill in attributes based on other attribute values. Multiply your length, width and height attributes to get a volume, or create a more complex calculation to forecast your margin. No more time wasted calculating values!
+
+This new action comes with a `round_precision` parameter to allow you to round the final result of your rule operation(s). For example, set the number of decimal places you need or remove them all.
```

Comment:
"decimal places" ? pour dire quoi exactement ? (je n'arrive pas à le traduire)

Reply:
Google definition: _decimal places: the position of a digit to the right of a decimal point._